android/src/main/java/nordpol/AndroidCard.java

```diff
@@ -22,17 +23,27 @@ private AndroidCard(IsoDep card) {
         this.card = card;
     }
 
+    /**
+     * Model names&numbers: 
+     * https://support.google.com/googleplay/android-developer/answer/6154891?hl=en 
+     */
+    private static boolean isSamsungS5() {
```

Comment:
This filter seems to be accurate over the model+device combinations in the document at the provided URL. But it would be good to have someone confirm this...